apps/prairielearn/elements/pl-checkbox/pl-checkbox.mustache

```diff
@@ -1,6 +1,6 @@
 {{#question}}
 {{#inline}}<span class="d-inline-block">{{/inline}}
-{{^inline}}<div class="d-block">{{/inline}}
+{{^inline}}<fieldset class="d-block"><legend class="sr-only">Checkbox options</legend>{{/inline}}
```

Comment:
Am I reading correctly that you're only rendering a `<fieldset>` if inline display isn't used?

Reply:
Yes. Updated to use the method above (`<span role="group" aria-label="...">) here so it still displays inline.